docs/semgrep-app/getting-started-with-semgrep-app.md

```diff
@@ -0,0 +1,280 @@
+</dl>
+
+### Signing in with GitLab
+
+1. Click the following link: [Sign into Semgrep](https://semgrep.dev/login?return_path=/manage/projects).
+2. Select **Sign in with GitLab**. You are redirected to the GitLab sign in page if you are not currently signed in.
+3. Sign in with your credentials.
+4. Click **Authorize**. See the GitLab documentation about [Authorized applications](https://docs.gitlab.com/ee/integration/oauth_provider.html#authorized-applications) to understand the scope of permissions requested by Semgrep.
+5. You are redirected back to Semgrep App.
+6. Click **Accept** to accept Semgrep's Terms of Service.
+7. Optional: Fill out the survey and click Complete or click Skip to omit this step.
+
+You are now signed in to Semgrep App.
+
+
+#### Permissions for GitLab
+
+Semgrep requires the following permissions (scopes) in order to function:
+
+
+<dl>
+    <dt>`openid`, `email`, and `profile`</dt>
+    <dd>Enables authentication of a session through OpenID.</dd>
+</dl>
+
+## Performing a scan
+
+Scanning is Semgrep's primary operation. When you first sign into Semgrep App, it uses a default ruleset selected to enforce best practices for a repository's framework and programming language. Future scans may be further tuned to an organization's specific practices.
+
+Semgrep App enables users to choose what findings prevent a pull or merge request (PR or MR) from merging into the repository. Setting these blocking and non-blocking rules is achieved through the Rule Board.
+
+
+### Adding a project
+
+A **project** is a repository from your GitHub or GitLab account that you add to Semgrep App for scanning. Semgrep App can run scans on many projects with rules set in the Rule Board. When you add a project, Semgrep scans it for the first time with pre-selected rules chosen based on the repository's language and framework.
+
+
+To add a project:
+
+1. Ensure you are signed in to Semgrep App.
+2. Click **Projects** on the left sidebar.
+3. Optional: If you do not see the repository you want to add in the **Projects **page of Semgrep app, follow the steps in the succeeding sections to ensure that Semgrep App can detect the repository.{{
+4. Click **Setup New Project**, and then select which repository provider Semgrep to integrate with.
+5. For **GitHub Actions**:
+    1. Click **Add CI Job **next to the name of the project to add.
+    2. Optional: If you do not see the repository you want to add, follow the succeeding guide on **detecting GitHub repositories**.
+    3. Click **Commit file**. This commits a `semgrep.yml` file containing pertinent scan parameters, such as schedule, what branch to scan on, and so on.
+    4. Semgrep App generates a token composed of a **Secret name** and **Secret value**. Copy and paste these into your repository's settings. Links are provided by Semgrep App.

```

Comment:
```suggestion
    4. Semgrep App generates a token composed of a **Secret name** and **Secret value**. Enter the token in your repository's settings. Semgrep App provides you links to follow.

```
See copy and paste in https://developers.google.com/style/word-list?hl=en
<img width="771" alt="Screenshot 2022-04-21 at 14 05 07" src="https://user-images.githubusercontent.com/26669307/164454392-38db9479-8940-4486-a0ce-e404e39b9af2.png">